consensus/state.go

```diff
@@ -1146,6 +1152,10 @@ func (cs *State) getEntropy(height int64) *types.ChannelEntropy {
 		if entropy.Height != height {
 			panic(fmt.Sprintf("Height mismatch found between entropy and the height of entropy requested! %v vs %v", entropy.Height, height))
 		}
+		if cs.nextAeonStartFunc != nil && entropy.Entropy.NextAeonStart == 0 {
+			entropy.Entropy.NextAeonStart = cs.nextAeonStartFunc(height)
+			cs.newEntropy[height] = entropy
+		}
```

Comment:
Consensus state fills in NextAeonStart inside the BlockEntropy by querying the dkgRunner.